app/src/main/java/com/hedvig/app/feature/settings/Language.kt

```diff
@@ -40,17 +40,10 @@ enum class Language {
     }
 
     @TargetApi(Build.VERSION_CODES.N)
-    private fun applyV24(context: Context?, locale: LocaleWrapper): Context? {
+    private fun apply(context: Context?, locale: LocaleWrapper): Context? {
         when (locale) {
             is LocaleWrapper.SingleLocale -> {
-                val unwrappedLocale = locale.locale
-                Locale.setDefault(unwrappedLocale)
-                if (context == null) {
-                    return null
-                }
-                val config = Configuration(context.resources.configuration)
-                config.setLocale(unwrappedLocale)
-                return context.createConfigurationContext(config)
+                return applySingleLocale(context, locale)
```

Comment:
Does this block also need an `updateResources`-call?

Reply:
yes it must call `context.resources.updateConfiguration(config, context.resources.displayMetrics)`